src/guide/production.md

```diff
@@ -69,3 +69,14 @@ To move your project to production, you will have to replace the evaluation invi
 </custom-tabs>
 
 To obtain a certificate for a production environment, please register [here](https://greenlight.blockstream.com/) and make sure to notify your LSP(s).
+
+## Production checklist
+There are some usecases where you need to verify that are implemented correctly. Here is a checklist that you can use to verify that your application is production ready.
+
+- **Closed channels**: Make sure you allow your users to redeem their onchain balance in case their channels were closed. see [Handling channel closures](guide/closed_channels.md)
```

Comment:
```suggestion
- **Closed channels**: Make sure you allow your users to redeem their onchain balance in case their channels were closed. See [Handling channel closures](closed_channels.md)
```

Reply:
Done